python/cugraph-pyg/cugraph_pyg/utils/imports.py

```diff
@@ -30,3 +30,67 @@ def package_available(requirement: str) -> bool:
             return False
 
     return True
+
+
+class MissingModule:
+    """
+    Raises RuntimeError when any attribute is accessed on instances of this
+    class.
+
+    Instances of this class are returned by import_optional() when a module
+    cannot be found, which allows for code to import optional dependencies, and
+    have only the code paths that use the module affected.
+    """
+
+    def __init__(self, mod_name):
+        self.name = mod_name
+
+    def __getattr__(self, attr):
+        raise RuntimeError(f"This feature requires the {self.name} " "package/module")
+
+
+def import_optional(mod, default_mod_class=MissingModule):
+    """
+    import the "optional" module 'mod' and return the module object or object.
+    If the import raises ModuleNotFoundError, returns an instance of
+    default_mod_class.
+
+    This method was written to support importing "optional" dependencies so
+    code can be written to run even if the dependency is not installed.
+
+    Example
+    -------
+    >> from cugraph.utils import import_optional
```

Comment:
```suggestion
    >> from cugraph_pyg.utils.imports import import_optional
```

These docstrings copied from `cugraph` should be updated as well.